services/maven-metadata/maven-metadata.tester.js

```diff
@@ -6,11 +6,12 @@ const { isVPlusDottedVersionAtLeastOne } = require('../test-validators')
 const t = (module.exports = new ServiceTester({
```

Comment:
Any reason this couldn't use `createServiceTester()`?

Reply:
I'm not sure why it wasn't used before, but I elected to leave this in place so I could explicitly set `pathPrefix` since the `maven-metadata` and `v` portions of the route had to be split across `base` and `pattern`. 

I could certainly switch this to `createServiceTester` though and add the leading `/v` to the tests

Reply:
I'm 👍 on switching, eager to keep paring down these exceptions.

Reply:
Done and done!